### PR TITLE
Fixes wsod on signal projects table

### PR DESCRIPTION
**Pull request by tillyw**
## Associated issues

fixes https://github.com/cityofaustin/atd-data-tech/issues/9403

## Testing
**URL to test:** 
https://moped-test.austinmobility.io/moped/projects

**Steps to test:**
add new signals project and open signals project view

---
#### Ship list
- [x] Code reviewed 
- [x] Product manager approved
- [ ] Added to [QA test script](https://docs.google.com/spreadsheets/d/1n_O6MLh9cwwPf57HUM394Ea-z9uuoEV1-QW4axNZXLE/edit#) if applicable

### CODE REVIEW

File: moped-editor/src/views/projects/newProjectView/NewProjectView.js [hunk 1]
```diff
@@ -153,6 +153,7 @@ const NewProjectView = () => {
           moped_proj_phases: {
             data: [
               {
+                phase_name: "potential",
```
Comment (chiaberry) [inline moped-editor/src/views/projects/newProjectView/NewProjectView.js:156]: was this creating another bug?

Comment (tillyw) [reply moped-editor/src/views/projects/newProjectView/NewProjectView.js:156]: yes, that's why i had to redeploy

Comment (tillyw) [reply moped-editor/src/views/projects/newProjectView/NewProjectView.js:156]: from what I understand, when you create a new project in the new project view without this default value, the default `phase_name` is null and it causes an error.

Comment (mateoclarke) [reply moped-editor/src/views/projects/newProjectView/NewProjectView.js:156]: a separate bug from https://github.com/cityofaustin/atd-data-tech/issues/9403? Or related?

Comment (tillyw) [reply moped-editor/src/views/projects/newProjectView/NewProjectView.js:156]: same bug. it didn't work until i added this line

**Review by mddilley — COMMENTED** [review]
This may be what Chia was talking about in dev sync about another way to do this, but I wanted to make sure that we eventually use `phase_id` here in the signals table since there is now a foreign key relationship. 

That way we don't have a straggler view/query that uses `phase_name` (which is the reason that `phase_name` was taken out of the payload of the `ADD_PROJECT` mutation). I could be wrong and maybe this is part of https://github.com/cityofaustin/atd-data-tech/issues/9350?

**Comment by mateoclarke** [comment]
I'm good with this but I liked Chia suggested, I think we should bookmark this and make sure to address with DB normalization in the future.

**Comment by chiaberry** [comment]
I linked to this PR on the normalization issue https://github.com/cityofaustin/atd-data-tech/issues/9350

**Review by amenity — COMMENTED** [review]
Fixed! Thanks for the speedy work, @tillyw !

**Pull request merged**